src/app/auth/features/user-manager-list/user-manager-list.component.html

```diff
@@ -0,0 +1,109 @@
+@if (usersAreLoading()) {
+  <p class="text-base">Loading users...</p>
+} @else {
+  @let columns = ['email', 'name', 'roles', 'status', 'registration_date', 'details'];
+  <div class="overflow-auto">
+    <table
+      mat-table
+      [dataSource]="source"
+      class="mat-elevation-z8"
+      matSort
+      #sort="matSort"
+    >
+      <ng-container matColumnDef="email">
+        <th mat-header-cell *matHeaderCellDef class="min-w-48" mat-sort-header>
+          Email
+        </th>
+        <td mat-cell *matCellDef="let user">
+          {{ user.email }}
+        </td>
+      </ng-container>
+
+      <ng-container matColumnDef="name">
+        <th mat-header-cell *matHeaderCellDef class="min-w-40" mat-sort-header>
```

Comment:
```suggestion
        <th mat-header-cell *matHeaderCellDef class="min-w-48" mat-sort-header>
```
We should avoid having names wrap